admin/src/App.js

```diff
@@ -2,11 +2,20 @@ import React, { Component } from 'react';
 import logo from './logo.svg';
 import './App.css';
 import HydraAdmin from 'api-platform-admin/lib/hydra/HydraAdmin';
+import platformVar from 'platformsh_variables';
+console.log(platformVar);
+let entrypoint = Object.keys(platformVar).find(url => url.startsWith('https://api---'));
```

Comment:
Ideally we wouldn't even hardcode the `api---` part. Also, it'll become `api.` once we have deployed Let's Encrypt on Platform.sh.